src/SourceBuild/tarball/content/prep.sh

```diff
@@ -13,10 +13,12 @@ usage() {
     echo "  --bootstrap    Build a bootstrap version of previously source-built packages archive."
     echo "                 This modifies the downloaded version, replacing portable packages"
     echo "                 with official ms-built packages restored from package feeds."
+    echo "  --withPrevious Download and extract the previous source-built SDK release specified in global.json."
```

Comment:
I feel like this hurts the UX of this for "normal" users like distro maintainers.  They should not be using this option.  It's presence can cause confusion.